src/graphql/main/schema.graphql

```diff
@@ -730,9 +730,9 @@ type Mutation {
   lnUsdInvoiceFeeProbe(input: LnUsdInvoiceFeeProbeInput!): SatAmountPayload!
   onChainAddressCreate(input: OnChainAddressCreateInput!): OnChainAddressPayload!
   onChainAddressCurrent(input: OnChainAddressCurrentInput!): OnChainAddressPayload!
-  onChainPaymentSend(input: OnChainPaymentSendInput!): PaymentSendPayload!
+  onChainPaymentSend(input: OnChainPaymentSendInput!): PaymentSendPayload! // This should allow for usd wallets
   onChainPaymentSendAll(input: OnChainPaymentSendAllInput!): PaymentSendPayload!
-  onChainUsdPaymentSend(input: OnChainUsdPaymentSendInput!): PaymentSendPayload!
+  onChainNoAmountUsdPaymentSend(input: OnChainUsdPaymentSendInput!): PaymentSendPayload!
```

Comment:
Not in love with calling this `NoAmount` however it is `NoAmount` in the sense that any btc address that has an amount specified will denominate that amount in sats and since this is denominated in cents it can only be used to send an amount that is chosen by the sender (a `NoAmount`).